student.rb

```diff
@@ -1,10 +1,11 @@
 require_relative 'person'
```

Comment:
![image](https://github.com/adamilare/school-library/assets/93039359/6cc23b10-d94a-4246-84a4-36b14ce76e59)

- [ ] It would be nice if you fix these links which are in your Table of Contents to make your readme file descriptive and professional because right now they are broken :fearful: Kindly remove them if you don't have any section regarding these links :+1: